Source/WebCore/Modules/mediacontrols/mediaControlsAdwaita.css

```diff
@@ -249,7 +249,7 @@ video::-webkit-media-controls-timeline.hidden {
 
 audio::-webkit-media-controls-volume-slider-container,
 video::-webkit-media-controls-volume-slider-container {
-    -webkit-appearance: media-volume-slider-container;
+    appearance: media-volume-slider-container;
```

Comment:
This value is no longer supported, so you can remove this line.

Reply:
Line has been removed.